src/Liip/RMT/Output/Output.php

```diff
@@ -117,6 +117,11 @@ public function writeEmptyLine($repeat = 1)
     // QuestionHelper does about the same as we do here.
     public function askQuestion(InteractiveQuestion $question, $position = null, InputInterface $input = null)
     {
+        if (class_exists('Symfony\Component\Console\Helper\QuestionHelper')) {
+            $helper = new \Symfony\Component\Console\Helper\QuestionHelper();
+            return $helper->ask($input, $this, $question->asSymfonyQuestion());
+        }
```

Comment:
Aren't you duplicate the code of line 127 here?  Why keeping the block starting at line 127?